src/plugins/github/cacheId.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import {type RepoId, repoIdToString} from "./repoId";
+
+/**
+ * A derived ID to reference a cache layer.
+ * Instances of this type will be:
+ * - Deterministic
+ * - Unique for this plugin
```

Comment:
These first two list items don’t really make sense. An instance of
`CacheId` is just a string, which isn’t deterministic or
nondeterministic, nor unique or shared.

It sounds like you want to say two separate things: first, that values
of this type (nit: “instances” suggests a class relationship, which
isn’t the case here) are lowercase and free of special characters, so
they can be portably used as file names; second, that `cacheIdForRepoId`
is a deterministic injective function. The second part is good to
document, but should be on the function, not the type.

Reply:
Moved comment. And added "Safe to use for filenames".

Reply:
Also replaced "instances" with returned CacheId's.